docs/user-guide/installation.md

```diff
@@ -2,8 +2,8 @@
 title: "Installation"
 description: "terraform-docs installation guide."
 menu:
-  docs:
-    parent: "user-guide"
+    docs:
```

Comment:
I think these whitespaces might have been added by the editor. Can you revert them back?

Reply:
yea, it's my editor. I didn't change the whitespaces there. Thanks for noticing them. I will go change it.